tests/fortran/f-igs.f

```diff
@@ -0,0 +1,58 @@
+      program figs
+      implicit none
+
+      include 'mpif.h'
+
+      integer npmax
+      parameter(npmax=16)
+
+      integer ierror,handle,hwait,np,me,i,neighbors,count
+      integer*8 id(npmax)
+
+      real*8 answer(npmax),u(npmax)
+
+      call mpi_init(ierror)
+      call mpi_comm_size(mpi_comm_world,np,ierror)
+      call mpi_comm_rank(mpi_comm_world,me,ierror)
+
+      count=1
+      if(me.gt.0) then
+        id(count)=me
+        count=count+1
+      endif
+      id(count)=me+1
+      count=count+1
+      if(me.lt.(np-1)) then
+        id(count)=me+2
+        count=count+1
+      endif
+
+      neighbors=count-1
+      call gs_setup(handle,id,neighbors,mpi_comm_world,np)
```

Comment:
@stgeke : In Fortran, we can't select between all reduce, pairwise and cystal router. So this
test will fail when the benchmarks choose crystal router as the method. Any workaround this?

Reply:
We might have to change the Fortran interface to come up with a workaround.